policyengine_us/parameters/gov/states/mt/tax/income/credits/elderly_homeowner_or_renter_credit/gross_income_sources.yaml

```
@@ -0,0 +1,22 @@
+description: Montana gross income sources for the calculation of elderly homeowner or renter credit.
+metadata:
+  unit: list
+  period: year
+  label: Monatna elderly homeowner or renter credit gross income sources
+  reference: 
+  - title: Montana 2022 Form2 Individual Income Tax Instructions, Page47, Gross Household Income
+    href: https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2022/12/Form-2-2022-Instructions.pdf#page=47
+  - title: 2022 Montana Code Annotated, Title 15, Chapter 30, Part 23, 15-30-2337 (9)
+    href: https://law.justia.com/codes/montana/2022/title-15/chapter-30/part-23/section-15-30-2337/
+
+values:
+  2009-01-01:
+    - adjusted_gross_income # Line1: federal adjusted gross income
+    - tax_exempt_interest   # Line2: exempt interest
```

space others similarly
```suggestion
    - tax_exempt_interest # Line 2: exempt interest
```